tests/test_mpscircuit.py

```diff
@@ -305,28 +303,39 @@ def test_MPO_conversion(highp, tfb):
     np.testing.assert_allclose(tensor4, tensor4_ref, atol=1e-12)
 
 
-@pytest.mark.parametrize(
-    "backend, dtype", [(lf("tfb"), lf("highp")), (lf("jaxb"), lf("highp"))]
-)
+@pytest.mark.parametrize("backend, dtype", [(lf("tfb"), lf("highp"))])
 def test_circuits_1(backend, dtype):
-    import time
-
-    begin = time.time()
     circuits = get_test_circuits(False)
-    print("time", time.time() - begin)
     do_test_canonical(circuits)
     do_test_wavefunction(circuits)
-    do_test_truncation(circuits, 0.9987293417932497, 0.999440840610151)
+    do_test_truncation(circuits, 0.9987310049079007, 0.9994452420583485)
     do_test_amplitude(circuits)
     do_test_expectation(circuits)
     external = external_wavefunction()
     do_test_fromwavefunction(external, 0.1185)
     do_test_proj(circuits, external)
     do_test_tensor_input(circuits)
     do_test_measure(circuits)
-    print("time", time.time() - begin)
 
 
-def test_circuits_2(highp):
+@pytest.mark.parametrize("backend, dtype", [(lf("tfb"), lf("highp"))])
+def test_circuits_2(backend, dtype):
     circuits = get_test_circuits(True)
     do_test_truncation(circuits, 0.9401410770899974, 0.9654331011546374)
+
+
+@pytest.mark.parametrize("backend, dtype", [(lf("tfb"), lf("highp"))])
+def test_circuits_jit(backend, dtype):
+    def expec():
+        mps = tc.MPSCircuit(N, split=split)
+        simulate(mps, False, check=False)
+        x = [0, 2]
+        y = [5, 3, 1]
+        z = [6, 8, 10]
+        exp = mps.expectation_ps(x=x, y=y, z=z)
+        return exp
+
+    expec_jit = tc.backend.jit(expec)
```

Comment:
what about test a parameterized circuit, i.e. ``expec`` with circuit weights as input, and test the composition of jit + AD?  i.e. `tc.backend.jit(tc.backend.value_and_grad(expec))`

Reply:
Sounds good